skrifa/src/patchmap.rs

```diff
@@ -3,18 +3,19 @@
 //! The IFT and IFTX tables encode mappings from subset definitions to URL's which host patches
 //! that can be applied to the font to add support for the corresponding subset definition.
 
+use core::ops::RangeInclusive;
 use std::collections::BTreeSet;
 
 use crate::GlyphId;
 use crate::Tag;
-use raw::tables::ift::DesignSpaceSegment;
 use raw::tables::ift::EntryFormatFlags;
 use raw::types::Uint24;
 use raw::{FontData, FontRead, FontRef};
 use read_fonts::{
     tables::ift::{EntryData, EntryMapRecord, Ift, PatchMapFormat1, PatchMapFormat2},
     ReadError, TableProvider,
 };
+use std::collections::HashMap;
```

Comment:
total nit but if this could be up with the other `std` imports that would satisfy my OCD